ios/ExposureProcessor.swift

```diff
@@ -123,19 +128,41 @@ public class ExposureProcessor {
         }
         self.scheduleCheckExposure()
     }
+
+    public func pause(_ resolve: @escaping RCTPromiseResolveBlock,
+                        rejecter reject: @escaping RCTPromiseRejectBlock) {
+        guard ENManager.authorizationStatus == .authorized else {
+            os_log("Not authorised so can't pause", log: OSLog.exposure, type: .info)
+            return reject("NOTAUTH", "Not authorised to start", nil)
+        }
+        let context = Storage.PersistentContainer.shared.newBackgroundContext()
+        Storage.shared.flagPauseStatus(context, true)
+        ExposureManager.shared.manager.setExposureNotificationEnabled(false) { error in
+            if let error = error as? ENError {
+                os_log("Error pausing./stopping notification services, %@", log: OSLog.exposure, type: .error, error.localizedDescription)
+                return reject("PAUSE", "Error pausing notification services", error)
+            } else {
+                os_log("Service paused", log: OSLog.exposure, type: .debug)
+                resolve(true)
+            }
+        }
+        
+    }
     
     public func stop(_ resolve: @escaping RCTPromiseResolveBlock,
                     rejecter reject: @escaping RCTPromiseRejectBlock) {
         guard ENManager.authorizationStatus == .authorized else {
             os_log("Not authorised so can't stop", log: OSLog.exposure, type: .info)
             return reject("NOTAUTH", "Not authorised to stop", nil)
         }
+        let context = Storage.PersistentContainer.shared.newBackgroundContext()
+        Storage.shared.flagPauseStatus(context, false)
```

Comment:
As per above comment

Reply:
Pause should be cleared by any call to stop. You are now longer in a paused state even if stop fails